roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -17,6 +17,12 @@ fsid = {{ fsid }}
 {% if common_single_host_mode is defined and common_single_host_mode %}
 osd crush chooseleaf type = 0
 {% endif %}
+{% if ceph_version not in ['jewel', 'kraken', 'luminous'] and containerized_deployment %}
```

Comment:
note for myself, perhaps it's better to do this for each daemon that is affected, eg: rgw and rbd-mirror

Reply:
Actually no because this will be overridden by https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-config/templates/ceph.conf.j2#L128-L132
This is good to go!